<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariableSection</class>
 <widget class="QDockWidget" name="VariableSection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>300</height>
   </rect>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>Variable Section</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QListWidget" name="LocalVariableList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>241</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="LocalLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Local Variables&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NewLocal">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>60</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteLocal">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>90</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditLocalName">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>120</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InsertBeforeLocal">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>180</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Before</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloneLocal">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>150</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clone</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isValueSetCheck">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Value set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LocalNameEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
